docs/user/alerting/action-types/servicenow.asciidoc

```diff
@@ -0,0 +1,72 @@
+[role="xpack"]
+[[servicenow-action-type]]
+=== ServiceNow action
+
+ServiceNow action type using [V2 Table API](https://developer.servicenow.com/app.do#!/rest_api_doc?v=orlando&id=c_TableAPI) to create ServiceNow incidents.
+
+[float]
+[[servicenow-connector-configuration]]
+==== Connector configuration
+
+ServiceNow connectors have the following configuration properties:
+
+Name::      The name of the connector. The name is used to identify a  connector in the management UI connector listing, or in the connector list when configuring an action.
+URL::    The from address for all emails sent with this connector, specified in `user@host-name` format.
```

Comment:
The description here is misleading, I believe a copy / paste mistake?

Reply:
the same here